pkg/asset/manifests/utils.go

```diff
@@ -21,6 +23,15 @@ type metadata struct {
 	Namespace   string            `json:"namespace,omitempty"`
 }
 
+type TlsCertificate struct {
+	privateKey  []byte
+	certificate []byte
+}
+
+const (
+	tlsExpirationDays = 365 * 2
```

Comment:
This is really excessive in the case of a bootstrap VM. It cannot normally run for more than 30-60 minutes. We can be generous and use a whole 1 day.

Reply:
Done